liquid-rust-tests/tests/neg/structs/default-option.rs

```diff
@@ -0,0 +1,16 @@
+#![feature(register_tool)]
+#![register_tool(lr)]
+
+// We need this right now to not deal with the enum `Some`
+
+#[lr::assume]
+#[lr::sig(fn(T) -> Option<T>)]
```

Comment:
You should be able to drop the `lr::sig` as this is the default signature, no?

Reply:
Yup, good catch!